src/remote-config.ts

```diff
@@ -2,6 +2,7 @@ import remoteConfig from '@react-native-firebase/remote-config';
 import {ENABLE_TICKETING, PRIVACY_POLICY_URL, CUSTOMER_SERVICE_URL} from '@env';
 
 export type RemoteConfig = {
+  contact_phone_number: string;
```

Comment:
We have used Firestore lately for things that doesn't need the capabilities of Remote Config, for example for URLs, and I think phone number is pretty much the same? Or do we envision to have some conditions on the phone number in Remote Config?

Reply:
It was decided at meetings to have this kind of information stored in the remote config to edit as if something goes wrong, The same applies to the phone, if we set it empty in the remote config then the button disappears. We need to involve the product owner and ask him, but that is what we decided on meetings.

Reply:
I don't think a non-technical product owner is the one that decides between having a phone number in RemoteConfig vs Firestore. The product owner states the need, and it is the developers that knows if Remote Config or Firestore is fit for the job.

If this is the need:
- Able to specify phone number outside of app release
- If no phone number specified, don't show button

… then I think Firestore is the correct place for it.

Reply:
The decision was not taken in a technical matter, it was based on the need to perhaps change texts or functionality as the first phase changes or requires actions.